.env.prod.example

```
@@ -0,0 +1,46 @@
+# Production Environment Configuration
+# Copy this to .env.prod for production deployment
+
+# Application Mode
+ENDO_API_MODE=production
+
+# Django Settings
+DJANGO_DEBUG=False
+DJANGO_ALLOWED_HOSTS=your-domain.com,localhost,0.0.0.0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DJANGO_ALLOWED_HOSTS key should go before the DJANGO_DEBUG key

(UnorderedKey)